.github/workflows/CI.yml

```diff
@@ -231,3 +231,43 @@ jobs:
             cd scipy/
             micromamba activate scipy-dev
             python dev.py test -t scipy.interpolate -v
+
+  misc:
+    name: Misc Test ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["macos-latest", "ubuntu-latest"]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: environment.yml
+          create-args: >-
+            lfortran=${{ env.LFORTRAN_VERSION }}
+            llvmdev=11.1.0
+
+      - uses: hendrikmuhs/ccache-action@main
+        with:
+          variant: sccache
+          key: ${{ github.job }}-${{ matrix.os }}
+
+      - name: Print installed packages information
+        shell: bash -e -x -l {0}
+        run: |
+            which lfortran
+            micromamba env list
+            micromamba activate lf
+            micromamba list
+            which lfortran
+
+      - name: Run LFortran Misc Tests
+        if: ${{ !startsWith(matrix.os, 'windows-2019') }}
+        shell: bash -e -x -l {0}
+        run: |
+            lfortran -g --debug-with-line-column tests/expr2.f90
+            ls -l
```

Comment:
Also, I think we need to ponder if the above is a good way of adding this test. Plus we also need to fix lfortran to exit with appropriate code when the exit code is > 255 (or < 0).

Reply:
It would be good to compare somehow the text that it prints.